pkg/apis/config/v1alpha1/resmgr/control/config.go

```diff
@@ -0,0 +1,26 @@
+// Copyright The NRI Plugins Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package control
+
+import (
+	"github.com/containers/nri-plugins/pkg/apis/config/v1alpha1/resmgr/control/cpu"
+)
+
+// +k8s:deepcopy-gen=true
+type Config struct {
+	Enable []string `json:"enable,omitempty"`
```

Comment:
In `common.control(.cpu)` I would prefer the opposite enable/disable logic. If there is a (non-empty) `cpu:...` definition, then it should be enabled by default.

Reply:
@askervin Reworked controller configuration and setup code. Eliminated configuration for explicitly enabling individual controllers by name. Instead, each controller now decides for itself whether it should be enabled or disabled and reports this back to the main controller during startup and reconfiguration. Updated CPU controller to disable itself if there are no CPU classes defined.